cli/src/main/java/org/jboss/as/cli/embedded/EmbedHostControllerHandler.java

```diff
@@ -217,58 +276,61 @@ protected void doHandle(CommandContext ctx) throws CommandLineException {
                 hostController = EmbeddedProcessFactory.createHostController(jbossHome.getAbsolutePath(), null, null, cmds);
             }
             hostController.start();
-            hostControllerReference.set(new EmbeddedProcessLaunch(hostController, restorer, true));
 
+            hostControllerReference.set(new EmbeddedProcessLaunch(hostController, restorer, true));
             ModelControllerClient mcc = new ThreadContextsModelControllerClient(hostController.getModelControllerClient(), contextSelector);
-            if (bootTimeout == null || bootTimeout > 0) {
-                long expired = bootTimeout == null ? Long.MAX_VALUE : System.nanoTime() + bootTimeout;
-
-                String localName = "master";
-                String status = "starting";
-
-                // read out the host controller name
-                final ModelNode getNameOp = new ModelNode();
-                getNameOp.get(ClientConstants.OP).set(ClientConstants.READ_ATTRIBUTE_OPERATION);
-                getNameOp.get(ClientConstants.NAME).set(Util.LOCAL_HOST_NAME);
-
-                final ModelNode getStateOp = new ModelNode();
-                getStateOp.get(ClientConstants.OP).set(ClientConstants.READ_ATTRIBUTE_OPERATION);
-                ModelNode address = getStateOp.get(ClientConstants.ADDRESS);
-                address.add(ClientConstants.HOST, localName);
-                getStateOp.get(ClientConstants.NAME).set(ClientConstants.HOST_STATE);
-                do {
-                    try {
-                        final ModelNode nameResponse = mcc.execute(getNameOp);
-                        if (Util.isSuccess(nameResponse)) {
-                            localName = nameResponse.get(ClientConstants.RESULT).asString();
-                            address.set(ClientConstants.HOST, localName);
-                            final ModelNode stateResponse = mcc.execute(getStateOp);
-                            if (Util.isSuccess(stateResponse)) {
-                                status = stateResponse.get(ClientConstants.RESULT).asString();
+
+            if (!emptyHost) {
+                if (bootTimeout == null || bootTimeout > 0) {
```

Comment:
Can these two be:

```
if (!emptyHost && (bootTimeout == null || bootTimeout > 0)) {
```
That will eliminate a whole lot of diff that's mostly whitespace coming from the extra indentation.